app/src/main/java/com/github/se/travelpouch/ui/home/TravelList.kt

```diff
@@ -76,25 +68,9 @@ fun TravelListScreen(
             }
       },
       bottomBar = {
-        NavigationBar(modifier = Modifier.testTag("navigationBarTravelList")) {
-          listOfDestinations.forEach { destination ->
-            NavigationBarItem(
-                onClick = {
-                  when (destination.title) {
-                    "Activities" -> navigationActions.navigateTo(Screen.TRAVEL_ACTIVITIES)
-                    "Calendar" -> navigationActions.navigateTo(Screen.CALENDAR)
-                  }
-                },
-                icon = { Icon(destination.icon, contentDescription = null) },
-                selected = false,
-                label = { Text(destination.title) },
-                modifier =
-                    Modifier.testTag(
-                        if (destination.title == "Activities") "navigationBarItemActivities"
-                        else "navigationBarItemCalendar"))
-            // modifier = Modifier.testTag("navigationBarItem"))
-          }
-        }
+        BottomNavigationMenu(
+            tabList = listOf(TopLevelDestinations.ACTIVITIES, TopLevelDestinations.CALENDAR),
```

Comment:
Here, there should not be an acativity tab. Because activities cannot be accessed from the list of travels. So what I suggest is to add a topLevelDestination called travels, and put it here instead of ACTIVITIES